test/e2e/util/util.go

```diff
@@ -148,6 +148,7 @@ func wrapErrorWithClusterIssuerStatusCondition(client clientset.ClusterIssuerInt
 
 // WaitForCertificateCondition waits for the status of the named Certificate to contain
 // a condition whose type and status matches the supplied one.
+// Deprecated: this function is not used anymore
```

Comment:
I suggest we remove all these unused functions. 
I don't expect any external projects to be importing packages from `cert-manager/test/...`